manipulation/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
Yes, I believe we should fully switch to c++11.
Otherwise we might end up with incompatible DSOs because of different boost implementations at some point.